code/game/objects/items/devices/radio/radio.dm

```diff
@@ -124,6 +124,18 @@
 	for(var/ch_name in channels)
 		secure_radio_connections[ch_name] = add_radio(src, GLOB.radiochannels[ch_name])
 
+/obj/item/radio/proc/refreshChannels() // slightly less destructive version of recalculateChannels() when you want to keep the current channels activated/deactivated
```

Comment:
make this snake_case since thats part of our contributing guidelines. also do a proper code doc

Reply:
refreshChannels no longer exists, so there's no need to doc it. though, some more extensive documentation was done on other procs in the radio.dm file.